build.sbt

```diff
@@ -14,13 +14,17 @@ lazy val commonSettings = Seq(
     "Cloudera" at "https://repository.cloudera.com/artifactory/cloudera-repos",
     "Twitter Maven" at "http://maven.twttr.com",
     "sonatype-snapshots" at "https://oss.sonatype.org/content/repositories/snapshots",
-    "Scalaz Bintray Repo" at "http://dl.bintray.com/scalaz/releases"
+    "Scalaz Bintray Repo" at "http://dl.bintray.com/scalaz/releases",
+    "sonatype-releases" at "https://oss.sonatype.org/content/repositories/releases/"
   )
 )
 
-lazy val root = project.in(file(".")).enablePlugins(PlayScala)
```

Comment:
so there is no root project. previous root has been changed into s2rest_play right?